terraform/modules/app-ecs-services/templates/alertmanager.tpl

```diff
@@ -47,4 +47,5 @@ receivers:
   - to: "${registers_zendesk}"
 - name: "dead-mans-switch"
   webhook_configs:
-  - url: "${dead_mans_switch_cronitor}"
+  - send_resolved: false
```

Comment:
Could we change the commit message that introduces this to explain importantly why we are adding this please